### ISSUE TITLE
One gamepad control setting for all platforms.

### ISSUE THREAD

**Issue by Roper89** [Swordfish90/Lemuroid#675]
Need profiles to customize gamepad controls for each platform.
It is inconvenient to reconfigure the controls every time due to the change of the console.
Example: control from "NES" is not compatible with "SMD"
NES: (B) - (A)
SMD: (A) - (B)

![image](https://github.com/Swordfish90/Lemuroid/assets/30005257/d9230e6a-f63a-40d9-98db-66d2c8b687e5)
![image](https://github.com/Swordfish90/Lemuroid/assets/30005257/5c310fc1-6e81-4011-be7c-8c8c0f2d78c2)

**Comment by rotcivetiel** [comment]
I agree. I love the app, but it broke me. I bought an M30 just because of the Lemuroid. Unfortunately the only one that worked with the six buttons was MD.emu

**Comment by rotcivetiel** [comment]
This is the M30 configuration for the Lemuroid with the six buttons working perfectly. Remembering that you have to activate the control with six buttons. Just open any game, press SELECT + START and go to the last item and select the six buttons.
![RDT_20231222_1623488066783627645886949](https://github.com/Swordfish90/Lemuroid/assets/140467703/03b95eed-dcc2-492d-affa-8f245310f5bc)
.
